### ISSUE TITLE
Migrate to Github Pages

### ISSUE THREAD

**Issue by pierpressure** [jnaulty/footplusplus-website#5]

**Comment by lambdaloop** [comment]
The problem is that AFAIK there is no way to forward the footplusplus.com domain to something with a / .
One solution is to create a separate footplusplus team account and put the website there.

**Comment by lambdaloop** [comment]
An alternative is to give contributors the credentials for AWS sub-account and allow them to use the deploy.sh script

**Comment by pierpressure** [comment]
yeah after reading your README the latter seems like a good possible solution

**Comment by pierpressure** [comment]
I'll keep this issue open for now as we continue testing how AWS will work
